dev/README_RELEASE_PROVIDER_PACKAGES.md

```diff
@@ -649,7 +648,11 @@ the artifact checksums when we actually release.
 The status of testing the providers by the community is kept here:
 <TODO COPY LINK TO THE ISSUE CREATED>
 
-You can find packages as well as detailed changelog following the below links:
+The issue is also the easiest way to see which code affecting PRs are included in the RC candidates.
```

Comment:
Nit: It would be better to rephrase `which code affecting PRs are included`

Reply:
Rephrased.